components/bl-transitions-component/src/collapse-left.js

```diff
@@ -6,29 +6,32 @@ const { cn } = BackendlessUI.CSSUtils;
 
 export function CollapseLeft(props) {
   const { component, eventHandlers, transitionsContainerPod, display, isContentLoaded, dynamicContent } = props;
-  const { classList, style, variants, duration } = component;
+  const { classList, style, variant, duration } = component;
   const { onMounted, onUnmounted, onEndAnimation, onStartAnimation } = eventHandlers;
 
   const [width, setWidth] = useState(0);
   const [initWidth, setInitWidth] = useState('');
 
   const rootRef = useRef();
-  const [element, setElement] = useState({ current: null });
+  const [element, setElement] = useState({});
 
   const isImagesLoaded = useImageLoad(element, dynamicContent);
   const isTransition = useTransition(element, display, duration, initWidth, width, 'width', onEndAnimation);
 
   useEffect(() => {
-    if (rootRef.current && !element.current) {
+    const readyToInitialTransition = rootRef.current && !element.current;
+
+    if (readyToInitialTransition) {
       setElement({ current: rootRef.current.firstElementChild });
```

Comment:
Same question as here https://github.com/Backendless/ui-builder-library/pull/518#discussion_r1196790726

Why we need property current?